lib/beaker-hostgenerator/data.rb

```diff
@@ -792,6 +800,14 @@ def osinfo
             'template' => 'ubuntu-1604-x86_64'
           }
         },
+        'ubuntu1604-POWER' => {
+          :general => {
+            'platform' => 'ubuntu-16.04-power8'
+          },
+          :abs => {
+           'template' => 'ubuntu-16.04-power8'
```

Comment:
Looks like we could use another space before `'template'` for proper indentation